npm-shrinkwrap.json

```diff
@@ -391,9 +391,9 @@
       "resolved": "https://registry.npmjs.org/isobject/-/isobject-2.1.0.tgz"
     },
     "jquery": {
-      "version": "2.2.4",
-      "from": "jquery@>=2.2.0 <3.0.0",
-      "resolved": "https://registry.npmjs.org/jquery/-/jquery-2.2.4.tgz"
+      "version": "3.1.0",
+      "from": "jquery@3.1.0",
+      "resolved": "https://registry.npmjs.org/jquery/-/jquery-3.1.0.tgz"
     },
     "jquery-ui": {
```

Comment:
Do we actually need jquery-ui? Looking at the internal Microsoft OSSSecurity tool it flags this as "use discouraged". What components require this?

Reply:
slickgrid requires it